indy_node/utils/node_control_utils.py

```diff
@@ -12,8 +25,15 @@
 class NodeControlUtil:
     @classmethod
     def run_shell_command(cls, command, timeout):
-        return subprocess.run(command, shell=True, check=True, universal_newlines=True,
```

Comment:
Did we remove `universal_newlines` and `check` intentionally?

Reply:
It was made to prevent automatic output stream decoding